### DISCUSSION TITLE
Can Lingua output ISO codes?

### DISCUSSION THREAD

**Discussion by mirix** [pemistahl/lingua-py#201] [Q&A]
Hello,

Can lingua output ISO codes or just language names?

Cheers,

Ed

**Comment by pemistahl** [comment]
This was just asked [yesterday](https://github.com/pemistahl/lingua-py/issues/198). It seems, I have to improve the documentation.

```python
>>> from lingua import Language
>>> Language.ENGLISH.iso_code_639_1
IsoCode639_1.EN
>>> Language.ENGLISH.iso_code_639_1.name
'EN'
```
